### DISCUSSION TITLE
Background handler not called after some days

### DISCUSSION THREAD

**Discussion by QuentinSc** [firebase/flutterfire#6042] [Q&A]
Hello,

1. Context
I'm using FirebaseMessaging.onBackgroundMessage() to register an handler in charge to display notification with flutter_local_notifications plugin.

2. Behavior

- Everything is fine except that users who didn't open app for days (dont know how many)  no longer see notification.

- The app is not manually  killed, just not in foreground for some days.

- If they reopen the app the can see notifications again.

- The FCM token dont change. 

3. Question
Is my background handler killed after some time ? Or it's a FCM particulary behavior ?

I can be the only one who wants notification working every day :( 

Thank you very much!

**Comment by designervoid** [comment]
Same, but I don't using `flutter_local_notifications_plugin`
https://github.com/FirebaseExtended/flutterfire/discussions/6543
